test/unit/logic/transaction.js

```diff
@@ -376,20 +376,24 @@ describe('transaction', () => {
 			});
 
 			transactionLogic.checkConfirmed(transaction, err => {
-				expect(err).to.not.exist;
+				expect(err).to.be.a('null');
 				done();
 			});
 		});
 
-		it('should return error for transaction which is already confirmed', done => {
+		it('should return true for transaction which is already confirmed', done => {
```

Comment:
Add another case for when there is a `Transaction#countById error` (after making the changes described in the first comment.